### ISSUE TITLE
Version 2.2.1 - AWSModel.h #import <Mantle/Mantle.h> file not found

### ISSUE THREAD

**Issue by DaveKerr** [aws/aws-sdk-ios#217]
Pod update to 2.2.1, build project and there is an error:
<img width="1280" alt="screen shot 2015-07-11 at 11 23 42 am" src="https://cloud.githubusercontent.com/assets/9383681/8634477/bc57ba86-27bf-11e5-99f9-cd0759a4a0ea.png">

Update all pods
Analyzing dependencies

CocoaPods 0.38.0.beta.2 is available.
To update use: `gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information see http://blog.cocoapods.org
and the CHANGELOG for this version http://git.io/BaH8pQ.

Downloading dependencies
Installing AWSCognito 2.2.1 (was 2.2.0)
Installing AWSCognitoSync 2.2.1 (was 2.2.0)
Installing AWSCore 2.2.1 (was 2.2.0)
Installing AWSS3 2.2.1 (was 2.2.0)
Generating Pods project
Integrating client project

**Comment by yosuke-matsuda** [comment]
Version 2.2.1 of `AWSModel.h` does not contain `<Mantle/Mantle.h>`. It looks like you need to clean your Xcode project cache files. If it does not help, you may want to delete `Podfile.lock` and `Pods/` and reinstall the pod.

**Comment by DaveKerr** [comment]
OK, I've updated cocoapods to the latest version.  I've deleted derived data directories and cleaned product.  I've deleted Podfile.lock and Pods directory.  I've executed 'pod install' which now downloads version 2.2.2.  I'm getting a linker error now, which I'm unable to find a solution:

Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_AWSCognitoCredentialsProvider", referenced from:
      objc-class-ref in AppDelegate.o
  "_OBJC_CLASS_$_AWSServiceConfiguration", referenced from:
      objc-class-ref in AppDelegate.o
  "_OBJC_CLASS_$_AWSServiceManager", referenced from:
      objc-class-ref in AppDelegate.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
